src/pages/terms.tsx

```diff
@@ -0,0 +1,1036 @@
+                            issued in any given month must be received by PostHog according to the payment terms
+                            specified in the invoice. Unpaid amounts are subject to a finance charge of 1.5% per month
+                            on any outstanding balance, or the maximum permitted by law, whichever is lower, plus all
+                            expenses of collection and may result in immediate termination of Service. Customer shall be
+                            responsible for all taxes associated with the Licenses Materials other than U.S. taxes based
+                            on PostHog’s net income.
+                        </p>
+                    </div>
+                    <div>
+                        <p>
+                            If you choose annual payments, we will send an invoice. Please pay on time or it could lead
+                            to late fees.&nbsp;
+                        </p>
+                    </div>
+                    <div>
+                        <p>
+                            5.3 Our fees do not include any taxes, levies, duties or similar governmental assessments of
+                            any nature, including, for example, value-added, sales, GST, use or withholding taxes,
+                            assessable by any jurisdiction whatsoever in relation to your purchases under this Agreement
+                            (collectively, the “Taxes”). You are solely responsible for paying all Taxes associated with
+                            your purchases hereunder. If we have a legal obligation to pay or collect Taxes for which
+                            you are responsible for under this Clause 4.3, we shall invoice you and you shall pay that
+                            amount to us unless you provide us with a valid tax exemption certificate authorized by the
+                            appropriate taxing authority. We shall calculate applicable Taxes based on your billing
+                            address as detailed on the relevant Order Form or Quote (it is your duty to inform us if
+                            Taxes should be assessed on a different address). You shall promptly notify us of any
+                            changes to any of your addresses specified in an Order Form or Quote. Taxes shall not be
+                            deducted from or set-off against the fees in the applicable Order Form or Quote.
+                        </p>
+                    </div>
+                    <div>
+                        <p>
+                            All our prices are without sales tax. Hey, terms and conditions can be pretty boring, so
+                            well done on reading these thoroughly! As a reward, please&nbsp;
+                            <Link href="/merch">claim a free legalhog sticker</Link>&nbsp;from our&nbsp;
```

Comment:
@fraserhopper Should the sticker thing be some kind of Printfection thing, or should we have a code here to redeem a free sticker?

Reply:
I'm just going to comment this bit out for now. We can re-add when we do the TP thing.